components/private/Snapshot/MDCSnapshotTestCase.h

```diff
@@ -25,7 +25,20 @@
 /**
  * This will call FBSnapshotVerifyView but first check for supported iOS versions. Additionally,
  * this will use UIGraphicsImageRenderer to render the view correctly (including shadows).
+ *
+ * Permits no changed pixels.
+ *
+ * @param view The view to use for snapshot comparison.
  */
 - (void)snapshotVerifyView:(UIView *)view;
 
+
+/**
+ * This will call FBSnapshotVerifyView but first check for supported iOS versions. Additionally,
+ * this will use UIGraphicsImageRenderer to render the view correctly (including shadows).
+ * @param view the view to use for snapshot comparison.
+ * @param tolerancePercent the percentage of pixels that can differ while still passing.
```

Comment:
_Tip:_ I think we should specify that it should be 0-1 here so we don't have to dig through the library itself.